lib/webdriveragent.js

```diff
@@ -64,6 +65,14 @@ class WebDriverAgent {
   }
 
   async launch (sessionId) {
+    let useExistingWebDriverAgent = !!this.webDriverAgentUrl;
+    if (useExistingWebDriverAgent) {
```

Comment:
no need to assign this a variable for a single use. let's do this instead:

```js
if (this.webDriverAgentUrl) {
```

since we know that a truthy value means someone has actually passed something in